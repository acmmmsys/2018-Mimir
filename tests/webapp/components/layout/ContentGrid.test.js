import React from 'react';
import ContentGrid from '../../../../webapp/components/layout/ContentGrid'
import renderer from 'react-test-renderer';

let wrapper;

test('ContentGrid does render', () => {
    wrapper = shallow(
        <ContentGrid>

        </ContentGrid>
    );
    expect(wrapper).toMatchSnapshot();
});
