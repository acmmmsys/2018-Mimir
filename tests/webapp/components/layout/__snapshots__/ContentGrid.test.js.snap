// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentGrid does render 1`] = `
<Grid
  className="grid-layout grid-content"
  columns={0}
  verticalAlign="middle"
/>
`;
